ui/scss/component/_file-render.scss

```diff
@@ -52,6 +52,9 @@
   .markdown-preview {
     height: 100%;
     overflow: auto;
+    background-color: #fff;
+    padding: var(--spacing-xlarge);
+    margin-top: var(--spacing-large);
```

Comment:
I don't think we should add this. No other news/blog site that I can think of has something like it. Is there a reason you wanted to add it? Also the `#fff` would need to be changed to a css variable so we can change the actual color for dark mode.